grafana-operator/grafana-operator
# Update Grafana.yaml

## Description

openshift-sar and openshift-delegate-urls needs to be namespaced in order to allow users with view roles to log into grafana
## Relevant issues/tickets


## Type of change




## Checklist


## Verification steps
try to log into grafana with a simple user with view role on the project. without the modification yuo will get : 2021/06/25 13:03:48 provider.go:613: 200 GET https://10.10.0.1/apis/user.openshift.io/v1/users/~ {"kind":"User","apiVersion":"user.openshift.io/v1","metadata":{"name":"a-lmercuri","uid":"2455b737-94cc-4b83-997e-05e0b29fb97e","resourceVersion":"71039558","creationTimestamp":"2021-05-04T14:19:30Z","managedFields":[{"manager":"oauth-server","operation":"Update","apiVersion":"user.openshift.io/v1","time":"2021-05-04T14:19:30Z","fieldsType":"FieldsV1","fieldsV1":{"f:fullName":{},"f:identities":{}}}]},"fullName":"mr.pink","identities":["Active Directory:TUVSQ1VSSSBMdWNh"],"groups":["system:authenticated","system:authenticated:oauth"]}
2021/06/25 13:03:48 provider.go:613: 201 POST https://10.10.0.1/apis/authorization.openshift.io/v1/subjectaccessreviews {"kind":"SubjectAccessReviewResponse","apiVersion":"authorization.openshift.io/v1","allowed":false}
2021/06/25 13:03:48 provider.go:464: Permission denied for mr.pink@cluster.local for check {"resource":"namespaces","scopes":[],"verb":"get"}
2021/06/25 13:03:48 oauthproxy.go:641: 10.130.2.7:59472 Permission Denied: user is unauthorized when redeeming token
2021/06/25 13:03:48 oauthproxy.go:438: ErrorPage 403 Permission Denied Invalid Account
